docs/docs/development-guides/contribution.md

```diff
@@ -12,9 +12,9 @@ nav_order: 2
 1. Create a new Branch.
 2. Make some changes and develop a cool new feature or fix a Bug.
 3. In order to commit run `npm run commit` to run the interactive Commit utility.
-![npm run commit image](/docs/assets/images/git-cz.png)
+![npm run commit image](https://github.com/T-Systems-MMS/phonebook/blob/master/docs/docs/assets/images/git-cz.png?raw=true)

```

Comment:
Kannst du hier irgendwo eine BaseURL definieren? 
Vielleicht gibt es auch einen weg die links dynamisch anzupassen

Reply:
https://travis.media/how-to-add-images-in-jekyll-posts-with-relative-links

Reply:
Thanks! I did tried so many things...